exporter/stackdriverexporter/config.go

```diff
@@ -39,6 +38,23 @@ type Config struct {
 	// Must be set programmatically (no support via declarative config).
 	// Optional.
 	GetClientOptions func() []option.ClientOption
+
+	TraceConfig  TraceConfig  `mapstructure:"trace"`
+	MetricConfig MetricConfig `mapstructure:"metric"`
+	NumOfWorkers int          `mapstructure:"number_of_workers"`
+}
+
+type TraceConfig struct {
+	BundleDelayThreshold time.Duration `mapstructure:"bundle_delay_threshold"`
+	BundleCountThreshold int           `mapstructure:"bundle_count_threshold"`
+	BundleByteThreshold  int           `mapstructure:"bundle_byte_threshold"`
+	BundleByteLimit      int           `mapstructure:"bundle_byte_limit"`
+	BufferMaxBytes       int           `mapstructure:"buffer_max_bytes"`
+}
```

Comment:
@james-bebbington not for this PR, but bundle options are overkill. We already provide queuing and batching in the collector, and ideally the exporter should not do more than that.